srcpkgs/apcupsd/template

```diff
@@ -0,0 +1,65 @@
+# Template file for 'apcupsd'
+pkgname=apcupsd
+version=3.14.14
+revision=1
+build_style=gnu-configure
+configure_args="
+ --sysconfdir=/etc/apcupsd
+ --with-pwrfail-dir=/etc/apcupsd
+ --with-upstype=usb --with-upscable=usb --enable-usb --with-dev=
+ --enable-modbus-usb
+ --enable-net --enable-pcnet
+ --enable-snmp
+ --disable-gapcmon
+ --with-nis-port=3551"
+hostmakedepends="autoconf automake"
+makedepends="libusb-compat libusb-compat-devel net-snmp net-snmp-devel man-pages man-pages-devel"
+depends="libusb-compat net-snmp nullmailer"
+conf_files="
+	etc/apcupsd/apcupsd.conf
+	etc/apcupsd/hosts.conf
+	etc/apcupsd/multimon.conf
+	etc/apcupsd/apcupsd.css
+	etc/apcupsd/changeme
+	etc/apcupsd/commfailure
+	etc/apcupsd/commok
+	etc/apcupsd/offbattery
+	etc/apcupsd/onbattery
+	etc/apcupsd/apccontrol"
```

Comment:
These need to absolute paths with the leading /, iirc. You could do `conf_files="/etc/apcupsd/*"` as well